src/ucs/datastruct/bitmap.h

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCS_BITMAP_H_
+#define UCS_BITMAP_H_
+
+#include <ucs/sys/compiler_def.h>
+#include <stdint.h>
+
+BEGIN_C_DECLS
+
+
+typedef uint64_t ucs_bitmap_word_t;
+
+
+#define UCS_BITMAP_BITS_IN_WORD (sizeof(ucs_bitmap_word_t) * 8)
+
+
+#define UCS_BITMAP_WORD_INDEX(_bit_index) ((_bit_index) / UCS_BITMAP_BITS_IN_WORD)
+
+
+#define _UCS_BITMAP_BIT_IN_WORD_INDEX(_bit_index) ((_bit_index) % \
+                                                   UCS_BITMAP_BITS_IN_WORD)
```

Comment:
pls add comments and move macro bodies to next line

Reply:
Do we need to add comments to internal macros (starting with `_`)?